jsonld/examples/package_sbom.json

```diff
@@ -0,0 +1,79 @@
+{
+    "@context": "https://raw.githubusercontent.com/spdx/spdx-3-serialization-prototype-playground/main/jsonld/spdx-3.0-context.json-ld",
+    "@graph": [
+        {
+            "type": "CreationInfo",
+            "spdxId": "_:creationinfo",
```

Comment:
`CreationInfo` should have an `@id` but not an `spdxId`, as it's not an `Element`

Reply:
It's very difficult to enforce that distinction. The context maps spdxId to @id and the model is self doen't even know what an spdxId is (because the ID is an intrisic property of RDF).

Realistically, it's very confusing to mix them. We should just pick either `spdxId` _or_ `@id`. I know spdxId is supposed to have some more intrinsic meaning, but TBH that meaning feels very artificially imposed to me with very little upside (from a user understanding perspective anyway; maybe there is something I'm missing)

Reply:
But there _is_ a difference: `spdxId` is globally unique, while `@id` may be a local identifier.

Reply:
Fair, but again I suspect enforcement of this is going to be difficult